### PR TITLE
Update to Upstream v0.7.0

### PR DESCRIPTION
**Pull request by dtfranz**
Currently released `operator-controller` does not yet use `catalogd@v0.8.0` so keeping this at the currently vendored version for now.

### CODE REVIEW

File: openshift/generate-manifests.sh
Comment (ncdc) [inline openshift/generate-manifests.sh:1]: Before merging: squash into `UPSTREAM: <carry>: Add OpenShift specific files` commit

**Review by everettraven — APPROVED** [review]
Looks good to me

**Comment by ncdc** [comment]
/lgtm
/approve

/override ci/prow/verify-commits
/override ci/prow/verify-commits-shared
(these never pass on rebase PRs)

**Pull request merged**